corehq/apps/userreports/views.py

```diff
@@ -248,5 +261,5 @@ def _shared_context(domain):
     return {
         'domain': domain,
         'reports': ReportConfiguration.by_domain(domain),
-        'data_sources': DataSourceConfiguration.by_domain(domain),
+        'data_sources': DataSourceConfiguration.by_domain(domain) + list(CustomDataSourceConfiguration.all()),
```

Comment:
Ha, good point, shouldn't be to hard to do now.

On 12 February 2015 at 15:30, Cory Zue <notifications@github.com> wrote:

> In corehq/apps/userreports/views.py
> <https://github.com/dimagi/commcare-hq/pull/5520#discussion_r24580004>:
>
> > @@ -248,5 +261,5 @@ def _shared_context(domain):
> >      return {
> >          'domain': domain,
> >          'reports': ReportConfiguration.by_domain(domain),
> > -        'data_sources': DataSourceConfiguration.by_domain(domain),
> > +        'data_sources': DataSourceConfiguration.by_domain(domain) + list(CustomDataSourceConfiguration.all()),
>
> would be nice to be able to partition these by domain eventually too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/5520/files#r24580004>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa